src/xrGame/ui/UIPdaWnd.cpp

```diff
@@ -125,6 +125,34 @@ void CUIPdaWnd::Init()
     UITabControl->SetAutoDelete(true);
     AttachChild(UITabControl);
     CUIXmlInit::InitTabControl(uiXml, "tab", 0, UITabControl);
+
+    constexpr std::tuple<pcstr, pcstr> socIds[] = 
+    {
+        {"0", "eptTasks"},
+        {"1", "eptMap"},
+        {"2", "eptDiary"},
+        {"3", "eptContacts"},
+        {"4", "eptRanking"},
```

Comment:
To prevent conflicts let's use different name.
```suggestion
        {"4", "eptStalkersRanking"},
```